syscall/syscall.csv

```diff
@@ -189,7 +189,6 @@
 "task_setcanceltype","sched.h","defined(CONFIG_CANCELLATION_POINTS)","int","int","FAR int *"
 "task_spawn","nuttx/spawn.h","!defined(CONFIG_BUILD_KERNEL)","int","FAR const char *","main_t","FAR const posix_spawn_file_actions_t *","FAR const posix_spawnattr_t *","FAR char * const []|FAR char * const *","FAR char * const []|FAR char * const *"
 "task_testcancel","sched.h","defined(CONFIG_CANCELLATION_POINTS)","void"
-"task_tls_alloc","nuttx/tls.h","CONFIG_TLS_TASK_NELEM > 0","int","tls_dtor_t"
```

Comment:
Do we need to add it to `libs/libc/libc.csv`?

Reply:
task_tls_alloc is an internal api shouldn't expose to user application directly.

Reply:
@pkarashchenko do you think my answer is reasonable?

Reply:
Yes. Sorry, for long turnaround time. I'm quite busy recent weeks